packages/apps/spaces/app/organizations/[id]/components/Tabs/panels/AccountPanel/RenewalForecast/RenewalForecast.tsx

```diff
@@ -110,22 +114,24 @@ export const RenewalForecast: FC<{
               fontSize='2xl'
               color={isAmountSet ? 'gray.700' : 'gray.400'}
             >
-              {isAmountSet ? formatCurrency(amount) : 'Unknown'}
+              {isAmountSet
+                ? formatCurrency(renewalForecast?.amount ?? 0)
+                : 'Unknown'}
             </Heading>
           </Flex>
         </CardBody>
-        {!!amount && updatedBy && (
+        {!!renewalForecast?.amount && renewalForecast?.updatedBy && (
```

Comment:
why those changes?

Reply:
because trying to destructure { something } from undefined results in a typeerror. Optional chaining solves that for the moment and I did not want to pursue the path of DTOs to handle this because DTOs are simply not doable if you think about the huge recursive graph that we have in the Timeline. So until we find a good scalable and maintainable pattern optional chaining is one way to solve it.

Reply:
or destructuring from null... same result.